source/Detector/Amptek/SGM/AmptekCommandManagerSGM.cpp

```diff
@@ -0,0 +1,213 @@
+#include "AmptekCommandManagerSGM.h"
+
+/// ===================== implementation of the static instance and methods ==================
+AmptekCommandManagerSGM* AmptekCommandManagerSGM::instance_ = 0;
+AmptekCommandManagerSGM* AmptekCommandManagerSGM::sddCommands(){
+	if(instance_ == 0)
+		instance_ = new AmptekCommandManagerSGM();
+
+	return instance_;
+}
+
+void AmptekCommandManagerSGM::releaseSDDCommands(){
+	if(instance_) {
+		delete instance_;
+		instance_ = 0;
+	}
+}
+
+/// ===================== implementation of the public methods ==================
+AmptekCommandManagerSGM::~AmptekCommandManagerSGM()
+{
+}
+
+QString AmptekCommandManagerSGM::requestStatusPacketHex() const{
+	return amptekCommand(ReqeustStatusPacket).hex();
+}
+
+//QString AmptekCommandManagerSGM::requestSpectrumHex() const{
```

Comment:
Are these (and their definitions in the .h) left commented for a reason, or can they be removed?